src/App.tsx

```diff
@@ -1,61 +1,122 @@
+import React, { FormEvent, useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { getUserById } from './utils';
+import { Todo } from './types/todo';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const preparedTodos = todosFromServer.map(todo => {
+  return {
+    ...todo,
+    user: getUserById(todo.id),
+  };
+});
+
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>(preparedTodos);
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [isTitleEmpty, setIsTitleEmpty] = useState(false);
+  const [isUserEmpty, setIsUserEmpty] = useState(false);
```

Comment:
this values are empty in the initial state, so it's not clear why did you set `false` values for them as the initial ones

will be better to rename it for better understanding that it's variables for errors, not just for checking if `title` and `user` are empty for now

Reply:
Thanks for suggestions, I'm fixing it